### PR TITLE
Fix several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by vikivivi** [comment]
I think the checks failed because suggestion is the same as misspell. Do you mean to suggest `encryption`?
```text
encription->encription
```
1. Shall we capitalise the word '`Fibonacci`? It is a name of a person.
```text
encription->encryption
fibonaacci->Fibonacci
```
2. Shall we add these additional misspell?
```text
adapations->adaptations, adoptions,
affinties->affinities
encompasing->encompassing
initializad->initialized
seamlessley->seamlessly
seeverity->severity
stategies->strategies
varously->variously
```
* https://github.com/search?q=adapations&type=code
* https://github.com/search?q=affinties&type=code
* https://github.com/search?q=encompasing&type=code
* https://github.com/search?q=initializad&type=code
* https://github.com/search?q=seamlessley&type=code
* https://github.com/search?q=seeverity&type=code
* https://github.com/search?q=stategies&type=code
* https://github.com/search?q=varously&type=code

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14156,6 +14169,7 @@ fhurter->further
 fials->fails
 fianite->finite
 fianlly->finally
+fibonaacci->fibonacci
```
Comment (isaak654) [inline codespell_lib/data/dictionary.txt:14172]: He was a famous mathematician, so:
```suggestion
fibonaacci->Fibonacci
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11937,6 +11949,7 @@ encoutners->encounters
 encouttering->encountering
 encrcypt->encrypt
 encripted->encrypted
+encription->encription
```
Comment (isaak654) [inline codespell_lib/data/dictionary.txt:11952]:
```suggestion
encription->encryption
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**